docs/undergrad.yaml

```diff
@@ -207,7 +207,7 @@ Bilinear and Quadratic Forms Over a Vector Space:
     orthogonal elements: 'bilin_form.is_ortho'
     adjoint endomorphism: 'bilin_form.left_adjoint_of_nondegenerate'
     Sylvester's law of inertia: 'https://en.wikipedia.org/wiki/Sylvester%27s_law_of_inertia'
-    real classification: 'https://en.wikipedia.org/wiki/Sylvester%27s_law_of_inertia'
+    real classification: 'quadratic_form.equivalent_one_zero_neg_one_weighted_sum_squared'
```

Comment:
If I remember correctly, this was discussed before, and left off because mathlib didn't have the _uniqueness_